configs/bugbounty/envoy-edge.yaml

```diff
@@ -0,0 +1,116 @@
+overload_manager:
```

Comment:
Please add this config and the other one to https://github.com/envoyproxy/envoy/blob/3750bc9342a5c5f35a44a1c4d91195265e6e53cc/configs/BUILD#L22 in some way so we run it through the config checker to check for syntax, deprecated config, etc.

I think we should point https://www.envoyproxy.io/docs/envoy/latest/configuration/best_practices/edge at these files, and have sufficient YAML comments in them, since these files are guaranteed to never go out of date and are mostly duplicated from the previous file. WDYT?

Reply:
Great idea, I have simplified the edge Envoy YAML to just a TLS listener and now it is included as the canonical example. This will be a nice way to avoid bit rot. Also added configs to config_test for regression.